tests/functional/test_inference_per_framework.py

```diff
@@ -1136,6 +1136,34 @@ def test_chat_api(self, nim_predictor, streaming, nchoices):
 
         assert "Boston! One of the oldest and most historic cities" in llm_response
 
+    @pytest.mark.parametrize("path", ["directAccess", "nim"])
+    def test_forward_http_request(self, path, nim_predictor):
+        base_url = f"{nim_predictor.url_server_address}/{path}"
+        response = requests.get(f"{base_url}/v1/models")
+        assert response.ok
+
+        # test HTTP GET through the "proxy" endpoint
+        response_data = response.json().get("data")
+        assert len(response_data) == 1
+        assert response_data[0].get("id") == "datarobot-deployed-llm"
+
+        from openai import OpenAI
+
+        client = OpenAI(
+            base_url=nim_predictor.url_server_address, api_key="not-required", max_retries=0
```

Comment:
This works? It doesn't need to be:
```suggestion
            base_url=base_url, api_key="not-required", max_retries=0
```